### ISSUE TITLE
Official PCKh on the validation set?

### ISSUE THREAD

**Issue by tangweibh** [anewell/pose-hg-train#41]
Hi Alejandro,

Thanks for sharing your codes! 

Since you have noted that the PCKh in the log files is not the official one, could you tell me how to obtain the official PCKh on the MPII validation set using your code, just like the PCKh results reported in your paper? 

Thanks,
Wei

**Comment by anewell** [comment]
All of the functions you need are in util/eval.lua. Specifically, if you have a set of predictions and their ground truth labels you can first call calcDists() to get normalized distances for each keypoint prediction. Then the function distAccuracy() will return the percentage of keypoints below a particular threshold.

These functions will ignore cases when no ground truth value is available for a particular keypoint.

You can also feel free to check out the official evaluation available from MPII: http://human-pose.mpi-inf.mpg.de

**Comment by tangweibh** [comment]
Got it! Thanks so much for your detailed instructions!
